src/Uno.UI/UI/Xaml/ResourceDictionary.cs

```diff
@@ -83,8 +83,15 @@ internal object Lookup(string key)
 		/// and can be removed as breaking change later.</remarks>
 		public bool Insert(object key, object value)
 		{
-			Set(new ResourceKey(key), value, throwIfPresent: false);
-			return true;
+			if (key is { })
```

Comment:
Does this match UWP behavior?

Reply:
It's not the same behavior, it's a temporary behavior to allow for the xaml trimming generated code to still work and not be too complex. I'll mention it in an issue.